components/RadioPlayer.vue

```diff
@@ -22,41 +30,33 @@
             <a href="#" class="text-secondary volumeshower" @mouseenter="showVolumeSlider = !showVolumeSlider">
```

Comment:
How come these are not part of the player modal? Maybe I'm misunderstanding what the Modal here means or what the role of the Modal is? Can  you link me to some place where I may dive deeper?

Reply:
this is not modal, but the volume slider, same as the current player on prod lahma

Reply:
Do you think the volume slider could be part of the player modal?